sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -333,24 +332,6 @@ export interface KeyVaultCertificateWithPolicy extends KeyVaultCertificate {
     readonly policy?: CertificatePolicy;
 }
 
-// @public
-export interface KVPollerLike<TState extends PollOperationState<TResult>, TResult> {
```

Comment:
@bterlson would probably say that we need to alias `KVPollerLike` to `PollerLike` for users who might be using the `KVPollerLike` interface for the output of `getCertificateOperation()` in their code. For example, below code that is valid today will fail to compile after the user updates to the newer version of the keyvault certificates library

```js
let result: KVPollerLike<CertificateOperationState, KeyVaultCertificateWithPolicy>;
result = await client. getCertificateOperation(...);
```

Reply:
I can do that! Thank you

Reply:
How does this look?

```ts
/**
 * Deprecated KeyVault copy of core-lro's PollerLike.
 */
export type KVPollerLike<TState extends PollOperationState<TResult>, TResult> = PollerLike<TState, TResult>;
```

Reply:
Am not familiar with how poller works, so @jonathandturner or @xirzec might be better suited to review this

Reply:
Gotcha! I will push this for now though.